xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -974,7 +974,12 @@
       { "name": "genre", "type": [ "null", { "$ref": "Array.String", "required": true } ], "default": null },
       { "name": "died", "$ref": "Optional.String" },
       { "name": "disbanded", "$ref": "Optional.String" },
-      { "name": "yearsactive", "type": [ "null", { "$ref": "Array.String", "required": true } ], "default": null }
+      { "name": "yearsactive", "type": [ "null", { "$ref": "Array.String", "required": true } ], "default": null },
+      { "name": "musicbrainzartistid", "$ref": "Optional.String" },
+      { "name": "sortname", "$ref": "Optional.String" },
+      { "name": "type", "$ref": "Optional.String" },
+      { "name": "gender", "$ref": "Optional.String" },
```

Comment:
Are `type` and `gender` freely choosable or are there specifically supported values? In case of the latter you could turn these into enums (same below).

Reply:
Like much of the music data `type` and `gender` are not codified and could conian any string, hence no enums.